tasks/request_files/schemas/input.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
We may want to revisit this later after we do the archiving workflow functionality to see if we can get away with using one translator for both the archive and recovery path. I would maybe write a card on the backlog for investigation to do that analysis for future work.

Reply:
Do you mean translator like the lambdas that construct this input? Or CMA?